templates/job-submitted.php

```diff
@@ -35,6 +35,30 @@
 				get_permalink( $job->ID )
 			)
 		);
+
+		$permalink = job_manager_get_permalink('job_dashboard');
+		$title = get_the_title((job_manager_get_page_id('job_dashboard')));
+
+		// If job_dashboard page exists but there is no title
+		if ( $permalink && empty($title) ) {
```

Comment:
Formatting around parens.

```suggestion
		if ( $permalink && empty( $title ) ) {
```